README.md

```diff
@@ -27,6 +27,7 @@ Any LTS version from 8.4.0 (with `--expose-http2`) or 8.8.1 without flags, onwar
 
 * clone the repository
 * `npm i` in the top level directory
+* `npm i -g lerna`
 * `lerna bootstrap` or `npx lerna bootstrap`
```

Comment:
Perhaps we should omit `lerna bootstrap` from this line instead and only have the `npx` one?